app/graphql/mutations/update_comment.rb

```diff
@@ -0,0 +1,16 @@
+module Mutations
+  class UpdateComment < BaseMutation
```

Comment:
Did you merge master? <img width="1440" alt="image" src="https://user-images.githubusercontent.com/35406008/206897410-79c49a0f-05ac-4e27-969e-542f580886e1.png">